packages/opc-feedback/README.md

```diff
@@ -0,0 +1,185 @@
+# opc-feedback Component 👋
+
+![Version](https://img.shields.io/badge/version-0.0.1-blue.svg?cacheSeconds=2592000)
+![Build Status](https://travis-ci.org/dwyl/esta.svg?branch=master)
+[![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://github.com/1-Platform/op-components/graphs/commit-activity)
+
+Feedback Component is used to collect feedback from end user.
+
+## Install
+
+```sh
+npm install --save @one-platform/opc-feedback
+```
+## Themes
+| Theme Name   | Example |
+|---------|------------------------------------------------------------------|
+| red (*default and fallback color*) | ```<opc-feedback theme="red"></opc-feedback>``` |
+| blue | ```<opc-feedback theme="blue"></opc-feedback>``` |
+
+### Color pallet
+
+| Color   | hex (default-fallback) |
+|---------|------------------------------------------------------------------|
+| --opc-feedback--BackgroundColor | ```#a30000``` |
+| --opc-feedback--TextColor | ```#fff``` |
+| --opc-feedback__submit--BackgroundColor |  ```#a30000``` |
+| --opc-feedback__submit--TextColor | ```#fff``` |
+
+## Properties
+```url``` - This allows to link your spa associated with feedback. By default ```/feedback``` is the url.
```

Comment:
is there only URL property that you're providing to your users? Add everything that is provided as a prop to user

Reply:
included the theme property and linked rhe usage reference